sdk/contentsafety/Azure.AI.ContentSafety/api/Azure.AI.ContentSafety.netstandard2.0.cs

```diff
@@ -163,7 +165,7 @@ public partial class TextBlocklist
     {
         internal TextBlocklist() { }
```

Comment:
What is the scenario for using the `CreateOrUpdateTextBlocklist` method?  Will the caller be expected to create a new instance of the TextBlocklist type, or will they always call `GetTextBlocklist` first?

If it is the former, TextBlocklist will need a public constructor that allows the caller to pass in the parameter required to create the URI.

If it is the latter, the operation should be renamed to remove `Create` from it, since it cannot be created new.

It can be helpful to write even a short, pseudo-code usage sample of what it looks like for an end-user to call a method on a client.  That can help show you what APIs are required and make sure you have all the pieces you need to implement the scenario end-to-end.

Reply:
My understanding is we should consider all the user cases unless it is completely invalid. The two scenarios are both reasonable to me. I'll add a public constructor for `TextBlocklist`.

As for the name of method, it comes from the spec user defined, so personally I think it's better to keep `CreateOrUpdateTextBlocklist`.